posts/2024-05-31-liberty-project-starter-guide-IntelliJ.adoc

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: "Developing a cloud-native Java application with the Liberty Starter, Liberty Tools, and IntelliJ IDEA"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/Ruilin-Ma
+author_github: https://github.com/Ruilin-Ma
+seo-title: Developing a cloud-native Java application with the Liberty Starter, Liberty Tools, and IntelliJ IDEA - OpenLiberty.io
+seo-description: Learn how to create a Liberty Maven project for a new cloud-native Java application by using the Open Liberty Starter, and to edit your application using the Liberty Tools in IntelliJ IDEA.
+blog_description: Learn how to create a Liberty Maven project for a new cloud-native Java application by using the Open Liberty Starter, and to edit your application using the Liberty Tools in IntelliJ IDEA.
+open-graph-image: https://openliberty.io/img/blog/liberty-starter.png
+open-graph-image-alt: Open Liberty starter
+---
+= Developing a cloud-native Java application with Liberty Tools in IntelliJ IDEA
+Ruilin Ma <https://github.com/Ruilin-Ma>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+:figure-caption!:
+//Blank line here is necessary before starting the body of the post.
+
+Given the rapid rate of technological change, the array of tools available, and the rising complexity of cloud-native applications, it can be difficult to know how to start creating a cloud-native Java application. This short post to leads you through creating a simple cloud-native Java application and editing it within an IDE, all within just 10 minutes! Read on to learn how to easily create a Liberty Maven project for a new cloud-native Java application by using the Open Liberty Starter and to edit your application using the Liberty Tools in IntelliJ IDEA.
```

Comment:
```suggestion
Given the rapid rate of technological change, the array of tools available, and the rising complexity of cloud-native applications, it can be difficult to know how to start creating a cloud-native Java application. This short post leads you through creating a simple cloud-native Java application and editing it within an IDE, all within just 10 minutes! Read on to learn how to easily create a Liberty Maven project for a new cloud-native Java application by using the Open Liberty Starter and to edit your application using the Liberty Tools in IntelliJ IDEA.
```